content/announcements/2024-12-10-contribfest-kubecon-eu-2025.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: contribfest-kubecon-eu-2025
+title: Join Us for ContribFest at KubeCon EU 2025 in London!
+description: The cert-manager maintainers are hosting a ContribFest event on 2nd April, 2025, and you are invited!
+date: "2024-12-10T09:00:00Z"
+---
+
+Are you heading to KubeCon EU 2025 in London this April? If so, we have some exciting news: the cert-manager maintainers are hosting a ContribFest event on 2nd April, 2025, and you are invited!
+
+As a newly graduated CNCF project, there's never been a better time to get involved with cert-manager and make an impact in the Kubernetes ecosystem.
+
+In this blog post, we will explain what ContribFest is and list first good issues to help you get started, whether you are joining us in London at KubeCon EU 2025 or participating from anywhere in the world.
+
+## What is ContribFest?
+
+ContribFest is an opportunity to join project maintainers and community contributors to explore good first issues, hunt bugs, discuss improvements, and pair program with maintainers to contribute directly to CNCF projects during a 90-minute session.
+
+At this year's cert-manager ContribFest, maintainers including CyberArk's Maël Valais and Richard Wall, and Zenior's Erik Godding Boye will be on hand to show users of all experience levels how to get involved with the cert-manager project.
+
+## Good First Issues
+
+- **[#6637](https://github.com/cert-manager/cert-manager/issues/6637)**: Enable a lint to help avoid accidental shadowing of package imports.
+- **[#6695](https://github.com/cert-manager/cert-manager/issues/6695)**: Update the Helm chart’s READMEs to mention usage in K3d clusters.
+- **[#6622](https://github.com/cert-manager/cert-manager/issues/6622)**: Improve logging clarity in ACME challenge solver.
+- **[#6732](https://github.com/cert-manager/cert-manager/issues/6732)**: Add a missing comment or doc string for a function in the E2E tests.
+- **[#5925](https://github.com/cert-manager/cert-manager/issues/5925)**: Use `readOnlyRootFilesystem: true` for all containers.
+- **[#5514](https://github.com/cert-manager/cert-manager/issues/5514)**: Venafi Issuer to read `caBundle` from ConfigMap or Secret.
+- **[#2943](https://github.com/cert-manager/cert-manager/issues/2943)**: Allow configuration of the keystore file name.
+- **[#1686](https://github.com/cert-manager/cert-manager/issues/1686)**: Support for Microsoft Active Directory Certificate Services.
+- **[#2744](https://github.com/cert-manager/cert-manager/issues/2744)**: Documenting resolution for DigitalOcean and HTTP01 "connection timed out" error.
+- **[#2888](https://github.com/cert-manager/cert-manager/issues/2888)**: Allow configuration of `MaxConcurrentChallenges`.
+- **[#1520](https://github.com/cert-manager/cert-manager/issues/1520)**: Ingress-shim: Skip updating Certificate resource if one already exists.
```

Comment:
This is a dump of the first good issues we have at the moment in cert-manager/cert-manager.

They might not be super interesting... If we get accepted, let's work further on curating "actually good" first issues :D